LiskHQ/lisk-sdk
# Fix DeepScan found issues - Closes #4707

### What was the problem?

DeepScan found 66 issues of different severity in the code

### How did I solve it?

By refactoring the code to avoid the issues

### How to manually test it?

- DeepScan should show none of the issues displayed in https://deepscan.io/dashboard/#view=project&tid=6759&pid=8869&bid=113509&subview=issues (take into account that DeepScan for some reason is still reporting the issues but you can add the solved issues by using the checkbox `Fixed` in the `status` pane)
- Protocol-specs folder issues are ok as they're not part of a node.


### Review checklist

- [x] The PR resolves #4707
